### DISCUSSION TITLE
How Grafana configure with production .NET Application?

### DISCUSSION THREAD

**Discussion by umeshshende** [grafana/grafana#53781] [Alerting]
Locally i configured and setup its working fine with jaeger and prometheus I want to know how to configure Grafana in Production? we are using Azure with .Net Application .

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
